### ISSUE TITLE
macOS: Close When Unfocused Not Working

### ISSUE THREAD

**Issue by andrewcincotta** [hluk/CopyQ#2337]
Current on the lastest version. Any ideas on how to get this to work?

**Comment by hluk** [comment]
Is there any change if you enable "Hide main window" in Layout tab in Preferences dialog?

**Comment by andrewcincotta** [comment]
> Is there any change if you enable "Hide main window" in Layout tab in Preferences dialog?

That's what I have enabled with the issue persisting. Did you push an update or something?

**Comment by hluk** [comment]
> > Is there any change if you enable "Hide main window" in Layout tab in Preferences dialog?
> 
> That's what I have enabled with the issue persisting. Did you push an update or something?

No, just wondering since CopyQ can decide to keep the window open if it does not detect tray and window-minimize functionality (option "Hide main window" is the workaround).

I guess CopyQ just does not get an event when it is deactivated.

**Comment by Chealer** [comment]
Can you please follow steps 10 to 13 from [that checklist](https://blog.michael.gr/post/2014-11-bug-reporting-checklist/#10-list-the-steps-to-reproduce-the-problem)? Ticket #1907 reports this when clicking on the desktop.

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
